lib/error.js

```diff
@@ -75,7 +76,11 @@ AppError.translate = function (response) {
     }
   }
   else if (payload.validation) {
-    error = AppError.invalidRequestParameter(payload.validation)
+    if (payload.message && payload.message.indexOf(messages.any.required) >= 0) {
+      error = AppError.missingRequestParameter(payload.validation.keys)
```

Comment:
The naming of vars and fields in `missingRequetParameter` suggest that it's expecting a single parameter name, not a list.  I think we should either pass in just the first missing name, or stringify it explicitly here.  Or maybe change `missingRequestParameter` to `missingRequestParameters` and make it explicitly take a list, but we'd have to check if anyone is depending on the details of that message structure in that response.